src/Sulu/Bundle/MediaBundle/Resources/config/services.xml

```diff
@@ -27,6 +27,7 @@
         <parameter key="sulu_media.twig_extension.media.class">Sulu\Bundle\MediaBundle\Twig\MediaTwigExtension</parameter>
         <parameter key="tmp">Sulu\Bundle\MediaBundle\Twig\MediaTwigExtension</parameter>
         <parameter key="sulu_media.video_thumbnail_generator.class">Sulu\Bundle\MediaBundle\Media\Video\VideoThumbnailService</parameter>
+        <parameter key="sulu_media.disposition_type.class">Sulu\Bundle\MediaBundle\Media\DispositionType\DispositionTypeService</parameter>
```

Comment:
we not longer create a parameter for class attributes to keep the container small write the class directly into the service definition.